### PR TITLE
Ohmyglb operator chart

### PR DESCRIPTION
**Pull request by ytsarev**
* Create ohmyglb chart including dependencies

Motivation

* Control operator and subchart config variables in
  a single place
* Handle use cases like image location in private registry
* More streamlined installation

### CODE REVIEW

**Comment by ytsarev** [comment]
@donovanmuller please provide early feedback. The PR requires more testing and love before the merge.

File: chart/ohmyglb/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application.
+appVersion: 1.16.0
```
Comment (donovanmuller) [inline chart/ohmyglb/Chart.yaml]: Would this be reset to reflect the release version when released?

File: chart/ohmyglb/values.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
+global:
+  ## Reference to one or more secrets to be used when pulling images
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ##
+  imagePullSecrets: []
+  # - name: "image-pull-secret"
+
+ohmyglb:
+  image: ytsarev/ohmyglb:v0.2.1
```
Comment (donovanmuller) [inline chart/ohmyglb/values.yaml]: Similar to Helm version, reset on first release?

File: chart/ohmyglb/values.yaml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      parameters: cluster.local
+    - name: loadbalance
+      parameters: round_robin
+    # Serves a /metrics endpoint on :9153, required for serviceMonitor
+    - name: prometheus
+      parameters: 0.0.0.0:9153
+    - name: forward
+      parameters: . /etc/resolv.conf
+    - name: etcd
+      parameters: absa.internal
```
Comment (donovanmuller) [inline chart/ohmyglb/values.yaml]: For future perhaps but refrain from using Absa in any configuration defaults

File: deploy/coredns/external-dns.yaml [hunk 4]
```diff
@@ -41,6 +41,8 @@ kind: ServiceAccount
 metadata:
   name: external-dns
   namespace: ohmyglb
+imagePullSecrets:
+- name: regcred
```
Comment (donovanmuller) [inline deploy/coredns/external-dns.yaml]: Would a pull secret be a Chart option that defaults to none. I.e. In general use perhaps image pull secrets would not be necessary

Comment (ytsarev) [reply deploy/coredns/external-dns.yaml]: @donovanmuller that's clearly a bug, thanks for the catch

File: chart/ohmyglb/values.yaml (hunk 3 above)
Comment (ytsarev) [reply chart/ohmyglb/values.yaml]: @donovanmuller `example.com` ?

File: chart/ohmyglb/Chart.yaml (hunk 1 above)
Comment (ytsarev) [reply chart/ohmyglb/Chart.yaml]: @donovanmuller argh, this number was automatically generated by `helm create..`

File: chart/ohmyglb/values.yaml (hunk 2 above)
Comment (ytsarev) [reply chart/ohmyglb/values.yaml]: @donovanmuller we can, but reset to what ? `v0.2.1` btw reflects semantics of actual development

File: chart/ohmyglb/values.yaml (hunk 3 above)
Comment (donovanmuller) [reply chart/ohmyglb/values.yaml]: Yeah 👍

**Pull request merged**